libraries/AP_DDS/README.md

```diff
@@ -253,6 +255,16 @@ requester: making request: ardupilot_msgs.srv.ModeSwitch_Request(mode=4)
 response:
 ardupilot_msgs.srv.ModeSwitch_Response(status=True, curr_mode=4)
 ```
+
+Call the takeoff service:
+
+```bash
+$ ros2 service call /ap/takeoff ardupilot_msgs/srv/takeoff "{alt: 10}"
```

Comment:
It's `ardupilot_msgs/srv/Takeoff` with a capital `T`, otherwise you get:
```
$ ros2 service call /ap/takeoff ardupilot_msgs/srv/takeoff "{alt: 10}"
The passed service type is invalid
```